documentation/index.md

```diff
@@ -423,25 +423,25 @@ component, and leaves out the `<span>` children of the `<Text>` components. The
 the `<App>` wrapper component, and the `<Text>` wrapper component.
 
 
-## Stateless components
-Because [stateless components](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions) can't be instantiated, `renderIntoDocument` won't return an instance back. 
+## Function components
+Because [function components](https://reactjs.org/docs/components-and-props.html#function-and-class-components) can't be instantiated, `renderIntoDocument` won't return an instance back. 
```

Comment:
Given that `when deeply rendered` [adds a wrapper for function components](https://github.com/bruderstein/unexpected-react/blob/0532142a9d181be29ea2d2e7285b89c201b28c93/src/assertions/deepAssertions.js#L91) behind the scenes, is this part still valid?